pkg/placement/placement-decision.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package placement
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/kubestellar/kubestellar/api/edge/v1alpha1"
+	"github.com/kubestellar/kubestellar/pkg/util"
+)
+
+// handlePlacementDecision syncs a placement-decision object with what is resolved by the placement-resolver.
+func (c *Controller) handlePlacementDecision(obj runtime.Object) error {
```

Comment:
`Controller` also has a `handleObject` method. The reader wonders why PlacementDecision objects need a special handler.

The problem here is using the word "handle" to mean two things: handle an event notification from an informer, and do the sync AKA reconcile work on an object reference pulled from the work queue. I strongly recommend using different words for these two in the func names. For example, this method might be called `syncPlacementDecision`.